### DISCUSSION TITLE
Rename `master` to `main`

### DISCUSSION THREAD

**Discussion by cpcloud** [ibis-project/ibis#3460] [General]
Seems like folks are expecting `main` to be the default everywhere now. Let's follow suit and rename our default branch to `main`.

@jreback Thoughts?

**Comment by cpcloud** [comment]
Doesn't seem to be much traction here. Closing for now.

**Comment by lostmygithubaccount** [comment]
should we make this change? particularly as ibis-substrait, ibisml, ibis-birdbrain, and any future projects in the org will default to main. any reason not to switch over at this point? GitHub should do a decent job at redirects and everything

**Reply by saulpw** [reply]
I vote no.  Any future projects can default to main but it's not worth one iota of friction to instigate a change like this that has no value and that no one asked for.

**Reply by lostmygithubaccount** [reply]
The value to me is in consistency and cognitive overhead -- while it's not a lot, I do have bash aliases that are hardcoded to `master` or `main` and I'd like to not deal with the differences (I recognize I could work around this, but Ibis is the last repo I regularly touch that uses `master`)

also from a new contributor or just casual user perspective, it's confusing for every other repo (Ibis Substrait, IbisML, Ibis Birdbrain, governance) to have a different default branch name

**Reply by gforsyth** [reply]
I'm in favor of the rename, also for general consistency and alias ergonomics.
AND it might be gnarly getting all the CI jobs updated accurately, so whomever wants to champion this should get ready for some YAMLgineering.

**Reply by NickCrews** [reply]
I'm +1 to rename
